Cannot build toolbox locally

Maven version:
```
Apache Maven 3.9.9 (8e8579a9e76f7d015ee5ec7bfcdc97d260186937)
Maven home: /usr/share/java/maven
Java version: 24.0.1, vendor: Arch Linux, runtime: /usr/lib/jvm/java-24-openjdk
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "6.13.2-arch1-1", arch: "amd64", family: "unix"
```
Following the instructions in DEVELOPING.md I ran `mvn install -Dmaven.test.skip=true` in the repository root. However, this resulted in the following build error:
>[Fatal Error] :135311:1935: JAXP00010003: The length of entity "[xml]" is "100,001" that exceeds the "100,000" limit set by "jdk.xml.maxGeneralEntitySizeLimit".
[ERROR] Failed to resolve target definition /mnt/data/ahelwer/src/tlaplus/java-tools/toolbox/org.lamport.tla.toolbox.product.product/TLAToolbox.target: Failed to load p2 metadata repository from location http://download.eclipse.org/releases/2018-09/201809191002: Unable to read repository at file:/home/ahelwer/.m2/repository/.cache/tycho/p2-repository-metadata/1895876646. JAXP00010003: The length of entity "[xml]" is "100,001" that exceeds the "100,000" limit set by "jdk.xml.maxGeneralEntitySizeLimit". -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException

If I run `mvn install -Dmaven.test.skip=true -Djdk.xml.maxGeneralEntitySizeLimit=0 -Djdk.xml.totalEntitySizeLimit=0` then I get a different error:
>[INFO] Resolving dependencies of MavenProject: tlatoolbox:org.lamport.tlatools:1.0.0-SNAPSHOT @ /mnt/data/ahelwer/src/tlaplus/java-tools/tlatools/org.lamport.tlatools/pom.xml
[ERROR] Cannot resolve target definition:
[ERROR]   Software being installed: javax.activation 1.1.0.v201211130549
[ERROR]   Missing requirement: javax.activation 1.1.0.v201211130549 requires 'java.package; javax.imageio 0.0.0' but it could not be found
[ERROR] 
[ERROR] Failed to resolve target definition /mnt/data/ahelwer/src/tlaplus/java-tools/toolbox/org.lamport.tla.toolbox.product.product/TLAToolbox.target: See log for details -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException

--- a/DEVELOPING.md
+++ b/DEVELOPING.md
@@ -54,7 +54,7 @@ The compiled `tla2tools.jar` will be output to `tlatools/org.lamport.tlatools/di
 Build & Test Toolbox IDE
 ------------------------
 Install the following dependencies to your path:
- * [Java Development Kit](https://adoptium.net/) version 11+
+ * [Java Development Kit](https://adoptium.net/) version 11 (newer versions will [likely cause build failures](https://github.com/tlaplus/tlaplus/issues/1162#issuecomment-2737943830))
  * [Apache Ant](https://ant.apache.org/) version 1.9.8+
  * [Apache Maven](https://maven.apache.org/) version 3.9.7+
 

